src/applications/simple-forms/21-0845/sass/0845-auth-disclose.scss

```diff
@@ -5,3 +5,29 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
 @import "../../../../platform/forms/sass/m-form-confirmation";
+
+div:has(.schemaform-sip-alert) {
```

Comment:
should this be applied to all save in progress? (could add in the html), or only our form? (css here)

Reply:
No, it's "custom" spacing just for this form.